src/build.d

```diff
@@ -0,0 +1,892 @@
+#!/usr/bin/env rdmd
+/**
+DMD builder
+
+Usage:
+  ./build.d dmd
+
+TODO:
+- add different ENABLE modes
+- add all posix.mak Makefile targets
+- support 32-bit builds
+- test on OSX
+- test on Windows
+- allow appending DFLAGS via the environment
+- test the script with LDC or GDC as host compiler
+*/
+
+version(CoreDdoc) {} else:
+
+import std.algorithm, std.conv, std.datetime, std.exception, std.file, std.format,
+       std.getopt, std.parallelism, std.path, std.process, std.range, std.stdio, std.string;
+import core.stdc.stdlib : exit;
+
+const thisBuildScript = __FILE_FULL_PATH__;
+const srcDir = thisBuildScript.dirName.buildNormalizedPath;
+shared bool verbose; // output verbose logging
+shared bool force; // always build everything (ignores timestamp checking)
+
+__gshared string[string] env;
+__gshared string[][string] flags;
+__gshared typeof(sourceFiles()) sources;
+
+void main(string[] args)
+{
+    int jobs = totalCPUs;
```

Comment:
This variable is never used. A primary feature of a build system is to facilitate parallel builds. Are there plans to implement such?

Reply:
It always was (though of course the user should have been able to regulate this with `-j`): 

https://github.com/wilzbach/dmd/blob/e417f0e0dfd3ea117609cd48a9de53695aaaa3ad/src/build.d#L244

This is still the case for master:

https://github.com/dlang/dmd/blob/9e877e08d9933449380721d85e29b0f71a567cae/src/build.d#L141-L142

https://github.com/dlang/dmd/blob/9e877e08d9933449380721d85e29b0f71a567cae/src/build.d#L1317-L1320